### ISSUE TITLE
Independent colors for text and background

### ISSUE THREAD

**Issue by earlyriser** [pedromassango/bottom_navy_bar#52]
Could it be possible to add controls for text and background colors separately? Specifically I'd want to have a black button with white text

**Comment by pedromassango** [comment]
Yes it is possible. But Icurrently have limited time for this project.

You can help by adding this feature if possible!

**Comment by mohak852** [comment]
Assing this to me I can help you with this Project

**Comment by mohak852** [comment]
Be more presize what you want

**Comment by pedromassango** [comment]
Hi @mohak852 
This will wrk by adding a nre property into `BottomNavyBarItem` called background color, that will be used as the background color alternative for the active color.

**Comment by antonio-pedro99** [comment]
closed by #90
